### ISSUE TITLE
Demo OCD creations failing

### ISSUE THREAD

**Issue by kavikumark** [satrd/poi_fusion_engine3#1]
At first time run I am providing proper postgres credentials and ocd_base is created properly, and first demo ocd db is ocd_valencia_demo created without any tables and data, 
in the console logs I could see postgres sql password prompt, it's surprising how it's printed in log files. 
Every time it's considering as firstTime=yes only.
Can you please look into it and resolve. Thank you very much for your contribution.

**Comment by benmomo** [comment]
If the 'valencia_demo' is not created properly (no data), then there is an error there or before, the initial script does not execute well and therefore the 'firstTime' parameter is not set to no. Can you see any hint in the log file? It is by default in debug mode, so every information is printed there for testing purposes (anyway I should not show the root password, you are right). Is the database postgis enabled? Maybe there is an error there; I cannot reproduce exactly the error. You may try to execute the scripts (SQL files in https://github.com/satrd/poi_fusion_engine3/tree/master/frontend/WebContent/config/dbScripts) with pgAdmin3 and see what's going on...

**Comment by kavikumark** [comment]
Thank you for quick response. I will try it and will keep posted what ever the result may be. Thanks once again.

**Comment by kavikumark** [comment]
From back end I am able to execute scripts properly for both demo OCDs. But it's not showing available OCDs in admin OCD list. it's showing only valencia_demo (as it was created using web itself earlier).
It's still considering as firstTime=yes and valencia_demo status is not able to initiaze. always staying as grey color even after multiple clicks. No debug logs also.

**Comment by kavikumark** [comment]
Finally I got it work by doing following steps.
1. Replaced my tomcat path in both demo ocd sql files (as it was mentioned in installation docs also)
2. Ran tomcat in foreground using /home/tomcat/bin/catalina.sh run
3. It prompted for password - Password for user postgres: Here I typed password
4. It asked for password twice, possibly for both demo OCD files

Everything is working as explained by "Benjamin Molina". Really thank you for your project and your contributions to the community.
